tests/pep695_py312.pyi

```diff
@@ -0,0 +1,5 @@
+type lowercase_alias = str | int  # Y042 Type aliases should use the CamelCase naming convention
+type _LooksLikeATypeVarT = str | int  # Y043 Bad name for a type alias (the "T" suffix implies a TypeVar)
+type _Unused = str | int  # Y047 Type alias "_Unused" is not used
```

Comment:
Worth adding a generic alias? The current code doesn't treat them specially, but it might be good to have some just so we know flake8-pyi doesn't choke on them.

```suggestion
type _Unused = str | int  # Y047 Type alias "_Unused" is not used
type _List[T] = list[T]

y: _List[int]
```

Reply:
Good shout

Reply:
Oh, now pyflakes is complaining because there hasn't yet been a pyflakes release with PEP-695 support (I think it might be fixed on the pyflakes `main` branch? Not sure)

Reply:
Oh that's annoying. Maybe we can do something silly for now like `type _List[T] = list`

Reply:
I have a weak preference for using an idiomatic snippet here, so I've worked around the pyflakes false positive instead of avoiding it

Reply:
Agree that's better!